Dockerfile.gateway-ui

```diff
@@ -0,0 +1,11 @@
+FROM node:lts
+WORKDIR /app 
+COPY . /app
+RUN yarn install
+RUN yarn build
+RUN npm install -g serve
+# The following env variables can be given at runtime:
+# - PORT
+# - REACT_APP_FM_GATEWAY_API
+# - REACT_APP_FM_GATEWAY_PASSWORD
+CMD yarn build && serve -s apps/gateway-ui/build/
```

Comment:
Similar to #65, we should ideally just do a simple find / replace rather than a full rebuild. This is fine for now though, and that issue can tackle the find / replace for both.